modules/custom/cu_core/cu_core.install

```diff
@@ -5,44 +5,24 @@
  * Install hooks for cu_core module.
  */
 
-/**
- * Implemenets hook_enable().
- */
-function cu_core_enable() {
-
-}
 
 /**
  * Implements hook_install().
  */
 function cu_core_install() {
 
-  // Check for environmental variables to install other cores.
-  $check = cu_core_environment_check();
+  $hosting = cu_core_get_hosting();
 
-  // If needed, exit install before something else happens.
-  if ($check['exit?']) {
-    return;
+  if ($hosting == 'travis') {
```

Comment:
could these be enabled in `cu_core_environment_check()`? Maybe call it `cu_core_enable_hosting_requirements()`. Since the environment is only asked for here, I think it is okay to couple the two until the environment is needed elsewhere.

Reply:
cu_core_enable_hosting_requirements makes sense, but cu_core_environment_check should probably be moved to express_environment_check so Express knows about some of the more common hosting configurations like Pantheon and Travis.  The each group's core can handle what to do if hosting on a known host vs. an unknown/default host.